EIPS/eip-1815.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: 1815
+title: Interface for blind auctions
+author: Albert Acebrón (@corollari)
+discussions-to: https://github.com/ethereum/EIPs/issues/1745
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-03-04
+---
+
+
+## Simple Summary
+Standard interface for contracts implementing blind auctions.
+
+## Abstract
+Currently there doesn't exist any standard interface/contract for blind auctions, creating a void that has been filled with many de-facto standards such as the one used by ENS[1] or the one used in the blind auction contract present in Solidity's documentation[2]. This ERC aims to remove this fragmentation by proposing a standard interface for blind auction contracts.
+
+## Motivation
+Many different implementations of blind auctions [1][2][3][4][5] have appeared in the last years, each one reinventing the wheel and implementing blind auctions in ways that are conceptually similar but with special quirks and different interfaces, thus making it impossible for a system to interact with all of these without accommodating all these differences. This ERC aims to solve that by standarizing an interface for blind auctions contracts, which will enable the creation of systems that can interact with any blind auctions without requiring specialized code.
+
+## Specification
+```solidity
+pragma solidity ^0.5.0;
+
+/// @title ERC-1815 Interface for blind auctions
+/// @dev See https://eips.ethereum.org/EIPS/eip-1815
+///  Note: the ERC-165 identifier for this interface is 0xe3594b02.
+interface ERC1815 /* is ERC165 */ {
+    enum Mode { Open, Auction, Owned, Forbidden, Reveal, NotYetAvailable }
+
+    event AuctionStarted(bytes32 indexed hash, uint registrationDate);
+    event NewBid(bytes32 indexed hash, address indexed bidder, uint deposit);
+    event BidRevealed(bytes32 indexed hash, address indexed owner, uint value, uint8 status);
+    event HashRegistered(bytes32 indexed hash, address indexed owner, uint value, uint registrationDate);
+
+    function startAuction(bytes32 _hash) external;
+    function startAuctions(bytes32[] calldata _hashes) external;
+    function newBid(bytes32 sealedBid) external payable;
```

Comment:
Can you explain the content of `sealedBid`?

Reply:
Yeah, I was planning on doing that when adding the NATspec comments. Haven't had the time available to sit down and do it tho.

Reply:
Ok! It is coming together nicely. only if an automated script could ask these questions 😉